### PR TITLE
Convert ProfileCardHeader and ProfileCardBody to flex

### LINKED ISSUE #47: Convert profile grid and profile card components to flex

### PR DESCRIPTION
**Pull request by unzunz**
Fix #47

### CODE REVIEW

File: app/assets/javascripts/components/profile_card.es6.jsx [hunk 1]
```diff
@@ -17,10 +17,11 @@ class ProfileCard extends Component {
   get styles() {
     return {
       container: {
-        display: 'inline-block',
+        display: 'flex',
+        flexDirection: 'column',
         position: 'relative',
-        width: '33%',
-        height: '250px',
+        width: '30%',
```
Comment (maxwolffe) [inline app/assets/javascripts/components/profile_card.es6.jsx]: Are we avoiding width now that we are using flex?

File: app/assets/javascripts/components/profile_card_head.es6.jsx [hunk 2]
```diff
@@ -15,23 +15,18 @@ class ProfileCardHead extends React.Component {
...
       },
       icon: {
-        position: 'relative',
-        float: 'right',
+        position: 'absolute',
+        top: '0',
+        right: '6',
         lineHeight: '40px',
-        marginRight: '5px',
       }
```
Comment (warrenshen) [inline app/assets/javascripts/components/profile_card_head.es6.jsx]: Trailing comma here!

File: app/assets/javascripts/components/profile_card_head.es6.jsx [hunk 3]
```diff
@@ -15,23 +15,18 @@ class ProfileCardHead extends React.Component {
...
-        marginLeft: '5px',
-        fontSize: '24px',
+        borderBottom: 'solid #D6D6D6 1px',
       },
       icon: {
-        position: 'relative',
-        float: 'right',
+        position: 'absolute',
+        top: '0',
+        right: '6',
```
Comment (maxwolffe) [inline app/assets/javascripts/components/profile_card_head.es6.jsx]: Add units (px)

**Comment by maxwolffe** [comment]
Just a few style comments. Otherwise, LGTM.

File: app/assets/javascripts/components/profile_card_head.es6.jsx (hunk 3 above)
Comment (warrenshen) [inline app/assets/javascripts/components/profile_card_head.es6.jsx]: We should probably put `'6px'` here.

File: app/assets/javascripts/components/profile_card.es6.jsx (hunk 1 above)
Comment (warrenshen) [inline app/assets/javascripts/components/profile_card.es6.jsx]: We're going to try to fix this but for now it's okay!

**Pull request merged**